models/evc.py

```diff
@@ -1407,10 +1407,13 @@ def handle_link_up(self, link):
             link(Link): Link affected by link.up event.
 
         """
-        if self.is_intra_switch():
-            return True
-
-        if self.is_using_primary_path():
+        if any(
+            (
+                self.is_enabled() and self.is_active(),
```

Comment:
@Ktmi, it makes sense, however, I'm not completely sure if for static EVCs if they shouldn't try to redeploy back to its primary path. I had an understanding that the general requirement of not trying to move/deploy an EVC after a link up was just for dynamic circuits. I do empathize with the fact that this had a lot of back and forth discussing this, and there's a lot of corner cases, so let's confirm this minor network requirement with @italovalcy. In a recent issue adjacent, Italo was confirming that the switchover to static primary was expected for instance https://github.com/kytos-ng/mef_eline/issues/394. Thanks.

Reply:
This got solved on https://github.com/kytos-ng/mef_eline/pull/383/commits/f56d236a90c5435f253557a9ed04dff42f297690